### ISSUE TITLE
Enqueue Failure error during a test with WDA and real iOS device

### ISSUE THREAD

**Issue by jeangb** [appium/appium#7641]
## The problem

Hi,
When I try to hit on a an object on a certain page using an xpath, I have a "Enqueue Failure: UI Testing Failure - Failure fetching attributes for element <XCAccessibilityElement" error on the appium server.

It will work if I use the accessibility id, but only a few objects on this page have an accessibility id.
I downloaded the last WDA from github. 
This is strange because the appium inspector (the one from the Appium-iOS-Inspector-master project) is able to spy on this page. 
This is a page on which the user can scroll up to get more datas.

I saw in a WDA issue that there was a corrective on it, however it doesn't work.
Here's my [XCUIElement+FBIsVisible.m](https://gist.github.com/jeangb/967252b461e709962796c1a26447004d) from the webdriveragent on my computer. 

Can you help me ?

## Environment

* Appium version (or git revision) that exhibits the issue: 1.6.3
* I use Xcuitest framework
* Desktop OS/version used to run Appium: Mac OS 10.11.16
* Node.js version (unless using Appium.app|exe): 3.10.10
* Mobile platform/version under test: iOS 9.3
* Real device or emulator/simulator: real device, iPhone 6 +
* Xcode 8.2.1
* Appium CLI or Appium.app|exe: Appium CLI

**Comment by jeangb** [comment]
Thanks a lot @mykola-mokhnach, I can't say how I had the wrong file in WDA...
